fhir-ig-carin-bb/src/test/resources/JSON/EOB1.json

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
do we make the examples accessible from the registry, or we need it in the test dir too?

Reply:
We do not make examples accessible from the registry. This example did not come from the IG package. It was in the IG github repo.